src/app/(proper_react)/(redesign)/(public)/HeroImage.tsx

```diff
@@ -291,13 +291,14 @@ export const HeroImageAll = ({
         </text>
         <text
           fill="#5B5B5B"
-          style={{ whiteSpace: "pre" }}
+          style={{ whiteSpace: "pre", width: "100%", textAnchor: "middle" }}
           fontFamily="var(--font-inter)"
           fontSize="21"
```

Comment:
Sorry, classic "offtopic" @pdehaan, but should we drop the `fontSize` to 20 to add a bit more safety around potential long locales?

Reply:
I don't see why not: fa458e876